app/controllers/admin/dashboard_controller.rb

```diff
@@ -2,4 +2,15 @@ class Admin::DashboardController < Admin::BaseController
   def show
 
   end
+
+  def user
+    @user = User.find(params[:id])
+  end
+
+  def update
+    order = Order.find (params[:id])
+    order.status = "Shipped"
+    order.save
```

Comment:
I think I did this a couple of times as well, surprised there's no way to both update and save